### PR TITLE
Formatting options for editor added

### PR DESCRIPTION
**Pull request by Aca-jov**
SDNTB-108

### CODE REVIEW

**Comment by ioanpocol** [comment]
please rebase

File: scripts/superdesk/editor2/editor.js [hunk 1]
```diff
@@ -933,6 +906,48 @@ angular.module('superdesk.editor2', [
             }
         }
 
+        function setEditorFormatOptions(editorConfig, editorFormat, scope) {
+            for (var i = 0; i < editorFormat.length; i++) {
```
Comment (petrjasek) [inline scripts/superdesk/editor2/editor.js]: this would be easier to read with forEach

**Comment by petrjasek** [comment]
:+1:

**Comment by pavlovicnemanja** [comment]
:+1:

**Pull request merged**